lucene/CHANGES.txt

```diff
@@ -11,6 +11,8 @@ New Features
 
 * LUCENE-9004: Approximate nearest vector search via NSW graphs
 
+* LUCENE-9694: New tool for creating a deterministic index (Haoyu Zhai)
```

Comment:
Probably we could also back-port this to Lucene 8.9, right?  We are not using any Lucene 9.x only APIs?

If so, can you move this CHANGES entry down to that section?  (Hmm, if it does not yet exist, you can create a skeletal one).

Reply:
Good point. I'll create an 8.9 entry instead